app/mailers/admin_mailer.rb

```diff
@@ -1,11 +1,9 @@
 class AdminMailer < ActionMailer::Base
   helper :subdomain
-  default template_path: 'devise/mailer'
-  default from: 'CServe'
-
+  default from: MAIL_ID
```

Comment:
Unnecessary constant.
It is better to define in `ApplicationMailer` which inherits from `ActionController::Base`. This constant would be available across all the mailers which are inheriting that `ApplicationMailer`.
This is similar to how controllers work.